go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/k8snetworkplumbingwg/sriov-network-device-plugin
 go 1.17
 
 require (
-	github.com/Mellanox/rdmamap v1.0.0
-	github.com/Mellanox/sriovnet v1.0.3
+	github.com/Mellanox/rdmamap v1.0.1-0.20220811152017-9f6303b40e3a
```

Comment:
can you bump to release version : https://github.com/Mellanox/rdmamap/tree/v1.1.0 ?